payjoin/src/send/mod.rs

```diff
@@ -408,6 +416,7 @@ fn serialize_url(
     fee_contribution: Option<AdditionalFeeContribution>,
     min_fee_rate: FeeRate,
     version: &str,
+    opt_in_to_optimistic_merge: bool,
```

Comment:
Would prefer to have a send::multi_party::serialize_url that encapsulates send::serialize_url to abstract these sepecifics away from the general case.